code/game/objects/structures/industrial_lift.dm

```diff
@@ -227,16 +230,36 @@ GLOBAL_LIST_EMPTY(lifts)
 		destination = get_step_multiz(src, going)
 	else
 		destination = going
+	if(istype(destination, /turf/closed/wall))
+		var/turf/closed/wall/C = destination
+		do_sparks(2, FALSE, C)
+		C.dismantle_wall(devastated = TRUE)
+		for(var/mob/M in urange(8, src))
+			shake_camera(M, 2, 3)
+		
+		playsound(C, 'sound/effects/meteorimpact.ogg', 100, TRUE)
 	if(going == DOWN)
 		for(var/mob/living/crushed in destination.contents)
 			to_chat(crushed, "<span class='userdanger'>You are crushed by [src]!</span>")
 			crushed.gib(FALSE,FALSE,FALSE)//the nicest kind of gibbing, keeping everything intact.
 	else if(going != UP) //can't really crush something upwards
-		for(var/obj/structure/anchortrouble in destination.contents)
-			if(!QDELETED(anchortrouble) && anchortrouble.anchored && (!istype(anchortrouble, /obj/structure/holosign)) && anchortrouble.layer >= GAS_PUMP_LAYER) //to avoid pipes, wires, etc
+		var/atom/throw_target = get_edge_target_turf(src, turn(going, pick(45, -45))) //finds a spot to throw the victim at for daring to be hit by a tram
+		for(var/obj/structure/victimstructure in destination.contents)
+			if(!QDELETED(victimstructure) && (!istype(victimstructure, /obj/structure/holosign)) && victimstructure.layer >= GAS_PUMP_LAYER) //to avoid wires
+				playsound(src, 'sound/effects/bang.ogg', 50, TRUE)
+				if(victimstructure.anchored && initial(victimstructure.anchored) == TRUE) //prevents you from wrenching lockers/crates to easily break them open
```

Comment:
Is the initial value of anchored really the best thing to be checking here? Don't some lockers  start out as anchored?

Reply:
Is there a better way to check against this? My main goal here is preventing people from anchoring lockers/crates to the track and as far as I'm aware, none of the lockers/crates that spawn on-station or can be ordered are anchored by default. Adding a typecheck specifically for lockers/crates seem a bit much and I'm not aware of any that start anchored.

Reply:
Just unanchor it and push with no damage, if it contayner.

Reply:
> 
> 
> Is there a better way to check against this? My main goal here is preventing people from anchoring lockers/crates to the track and as far as I'm aware, none of the lockers/crates that spawn on-station or can be ordered are anchored by default. Adding a typecheck specifically for lockers/crates seem a bit much and I'm not aware of any that start anchored.

Sorry for leaving this so long, I guess things will become clear if this is a proper issue, smells a bit but what can you do